### PR TITLE
Clear confusion over recaptcha key names

### PR DESCRIPTION
**Pull request by PhilETaylor**
On the Google webpage when you create your Private/Public keys for the reCaptcha service, Google dont call them Public/Private anymore, they call them your Site and Secret keys.

This PR is the simple language change (only in the en-GB) for these two:

Private Key - becomes - Secret Key
Public Key - becomes - Site Key

screenshot 2015-01-02 23 10 20

screenshot 2015-01-02 23 27 24

### CODE REVIEW

**Comment by PhilETaylor** [comment]
Paging @brianteeman as he knows more about how I need to get language strings translated etc..

**Comment by brianteeman** [comment]
Phil check the NEW plugin in staging called captcha NoCaptcha (iirc) and
the discussion on that

**Comment by PhilETaylor** [comment]
Looks like noCaptcha also needs this language change then...

**Comment by brianteeman** [comment]
Ok I will double check in the morning. Not at my pc

**Comment by PhilETaylor** [comment]
Let me post my updated comment so you get as an email :-) 

>Looks like noCaptcha has this change already, but if the Recaptcha plugin is going to ship as well as the nocaptcha plugin then this is still needed

**Comment by brianteeman** [comment]
Simple fix - setting RTC

**Comment by infograf768** [comment]
This has to be squashed before merging.

**Comment by PhilETaylor** [comment]
er - how do I do that?

**Comment by zero-24** [comment]
>er - how do I do that?

The commiter bevor comitting :smile:

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini [hunk 1]
```diff
@@ -7,9 +7,9 @@ PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION="This CAPTCHA plugin uses the reCAPTCHA se
 PLG_CAPTCHA_RECAPTCHA="Captcha - ReCaptcha"
 
 ; Params
-PLG_RECAPTCHA_PUBLIC_KEY_LABEL="Public Key"
+PLG_RECAPTCHA_PUBLIC_KEY_LABEL="Site key"
 PLG_RECAPTCHA_PUBLIC_KEY_DESC="Used in the JavaScript code that is served to your users. See the plugin description for instructions on getting a public key."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini]: @PhilETaylor can you change also here `public key` to `site key`?

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini [hunk 2]
```diff
@@ -7,9 +7,9 @@ PLG_CAPTCHA_RECAPTCHA_XML_DESCRIPTION="This CAPTCHA plugin uses the reCAPTCHA se
...
-PLG_RECAPTCHA_PRIVATE_KEY_LABEL="Private Key"
+PLG_RECAPTCHA_PRIVATE_KEY_LABEL="Secret key"
 PLG_RECAPTCHA_PRIVATE_KEY_DESC="Used in the communication between your server and the ReCaptha server. Be sure to keep it a secret. See the plugin description for instructions on getting a private key."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini]: @PhilETaylor can you change also here `private key` to `secret key`?

**Comment by zero-24** [comment]
@PhilETaylor can you fix the names also here:

https://github.com/PhilETaylor/joomla-cms/blob/ClearConfusionOverReCaptchaKeyNames/administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini#L24-25
and here
https://github.com/PhilETaylor/joomla-cms/blob/ClearConfusionOverReCaptchaKeyNames/administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini#L29-30

**Comment by brianteeman** [comment]
Can this be committed please as I would like to submit a PR to resolve #5595 but it will conflict <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5593">issues.joomla.org/joomla-cms/5593</a>.</sub>

**Comment by zero-24** [comment]
@brianteeman 

can you include into your new patch this two strings that still calls `private` and `public` Key?

https://github.com/joomla/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini#L11
and
https://github.com/joomla/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini#L13

Thanks.

**Comment by brianteeman** [comment]
@zero24 this has been committed by Michael. Did I miss something

**Comment by brianteeman** [comment]
@zero-24 please see #5656 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/5593">issues.joomla.org/joomla-cms/5593</a>.</sub>

**Pull request closed without merging**